tgui/packages/tgui/interfaces/BlackMarketUplink.tsx

```diff
@@ -3,8 +3,39 @@ import { AnimatedNumber, Box, Button, Modal, Section, Stack, Tabs } from '../com
 import { formatMoney } from '../format';
 import { Window } from '../layouts';
 
+type BlackMarketUplinkContext = {
+  categories: string[];
+  markets: Market[];
+  items: Item[];
+  money: number;
+  viewing_market: string;
+  viewing_category: string;
+  buying: boolean;
+  ltsrbt_built: boolean;
```

Comment:
I believe we use `BooleanLike` when handling booleans passed in from byond (which are `1` or `0`, and not `true` or `false`)

Reply:
FUCK